source/includes/_fiat_to_crypto_flow.md

```diff
@@ -0,0 +1,236 @@
+# Fiat to Crypto flow
+
+This section describes the endpoints used in the fiat to crypto flow. 
+
+## Create Fiat to Crypto Invoice
+
+> Example Call
+
+```shell
+curl -X POST 'https://api.onramp.ltd/rpc/create_fiat_to_crypto_invoice' \
```

Comment:
I see we will have a new endpoint, it shouldn't be a problem since ZoomX would need to readapt their flow anyway.

Reply:
> I see we will have a new endpoint, it shouldn't be a problem since ZoomX would need to readapt their flow anyway.

Hm, technically we can have same create_ingress_invoice endpoint but with a slightly different payload and under the hood call different SQL endpoints.

Reply:
Yes, we may overload the create_ingress_invoice with other internal processing mechanics. Let me know should we proceed with this option.